lib/github.js

```diff
@@ -86,10 +86,27 @@ async function getPullRequestInfo({payload}) {
         number: payload.issue.number,
       },
     )
-    .then(res => ({
-      ...get(res, 'repository.pullRequest'),
-      repoName: payload.repository.full_name,
-    }));
+    .then(async res => {
+      const result = {
+        ...get(res, 'repository.pullRequest'),
+        repoName: payload.repository.full_name,
+      };
+
+      // graphql api returns `null` for repos the app isn't installed on,
```

Comment:
Womp, that's annoying.

Reply:
The problem is that the app *is* installed on the repo the pull request is in, but it also needs to get the name of the HEAD repo (which in some cases is a fork, which the app *isn't* installed on)

Reply:
Ah, got it.  Thanks for the clarification.  Here's hoping that GitHub doesn't deprecate the REST API anytime soon